### ISSUE TITLE
Add SLES15 to the e2e-testing matrix of linux vms we test against

### ISSUE THREAD

**Issue by EricDavisX** [elastic/e2e-testing#1733]
We would like to cite official support for this and so manual testing is done and has revealed no immediate risk.  The plan is to add this to the e2e-testing framework hosts list for Agent tests.

in support of this ticket:
https://github.com/elastic/beats/issues/27230

**Comment by jlind23** [comment]
Following the coding dojoj. Adding SLES15 may be as simple as adding the right image just there:
https://github.com/elastic/e2e-testing/blob/main/.ci/.e2e-tests.yaml

**Comment by jlind23** [comment]
@adam-stokes @mdelapenya where can i can the right image name (ami-***) ?

**Comment by EricDavisX** [comment]
@jlind23 I'm assigning you, and removing myself but want to stay involved on this topic.

- [ ] I think that question is relevant!  We should identify that general process as one point. 
- [ ] confirm content in recording / meeting notes is complete that we know where to put it.  :)

Final question to confirm is if we needed mechanisms in place to dictate which OSes should be covered in given frequency (and in PR CI) before we start adding lots more configurations to the list?  That was mentioned, but I'm unsure if it has a separate ticket that is being followed or if it is indeed really a blocker to adding more configurations (we could just go for it)
